examples/00-basic-examples/Circuit_Example.py

```diff
@@ -1,108 +1,112 @@
 """
-
 Circuit Example Analysis
---------------------------------------------
-This tutorial shows how you can use PyAedt to create a project in
-in NEXXIM Circuit and run a simulation
+------------------------
+This example shows how you can use PyAEDT to create a Circut design 
+and run a Nexxim time-domain simulation.
 """
 # sphinx_gallery_thumbnail_path = 'Resources/circuit.png'
 
 from pyaedt import Circuit
 from pyaedt import Desktop
 import os
+
 ###############################################################################
-# Launch Desktop and Circuit
-# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-# This examples will use AEDT 2021.1 in Graphical mode
+# Launch AEDT and Circuit.
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# This examples launches AEDT 2021.1 in graphical mode.
 
-# This examples will use SI units.
+# This examples uses SI units.
 
 desktopVersion = "2021.1"
 
 ###############################################################################
-# NonGraphical
-# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-# Change Boolean to False to open AEDT in graphical mode
+# Launch AEDT in non-graphical mode.
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Change the Boolean parameter ``NonGraphical`` to ``False`` to launch AEDT in 
+# graphical mode.
 
 NonGraphical = True
 NewThread = True
 
 ###############################################################################
-# Launch AEDT and Circuit Design
-# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-# Desktop Class initialize Aedt and start it on specified version and specified graphical mode. NewThread Boolean variables defines if
-# a user wants to create a new instance of AEDT or try to connect to existing instance of it
+# Launch AEDT and Circuit.
```

Comment:
Since these are headers, we should remove the period for all of these.  It's effectively a title.